synapseclient/__main__.py

```diff
@@ -1483,7 +1493,7 @@ def build_parser():
         type=str,
         help="Output the provenance record in JSON format",
     )
-    parser_get_provenance.set_defaults(func=getProvenance)
+    parser_get_provenance.set_defaults(func=get_activity)
```

Comment:
Got it - I will add in `get-activity` for the CLI.

What is the way to mark these are deprecated? I can add information into the help message like:
`help="show provenance records. This is *deprecated* - Swap to using `get-activity` instead."`

But is that sufficient?

Reply:
ok great - I am going to leave this calling `getProvenance`. In the console it prints out the warning:
`DeprecationWarning: Call to deprecated function (or staticmethod) getProvenance. (deprecated and replaced with `get_activity`) -- Deprecated since version 3.1.0.`